### PR TITLE
Revert "Freezers and heaters now drop the correct materials"

### LINKED ISSUE #28808: Cannot add last cable piece to finish a Freezer/Heater
Building a freezer or heater like normal. Cannot add last piece. Error message that says "You cannot add that to the machine" while shift+click inspect text says "It requires 1 cable piece."

### PR DESCRIPTION
**Pull request by nicbn**
Reopens #28560
Fixes #28808

#28808 is more critical than #28560.
Also, #28560 will be part of the #28810 issue.

### CODE REVIEW

**Comment by optimumtact** [comment]
it's not that critical, just fix it properly

**Comment by nicbn** [comment]
Gonna fix it later.

File: code/modules/atmospherics/machinery/components/unary_devices/thermomachine.dm [hunk 1]
```diff
@@ -32,18 +32,17 @@
...
-/obj/item/weapon/circuitboard/machine/thermomachine/Initialize()
-	. = ..()
-	if(!build_path)
-		if(prob(50))
-			name = "Freezer (Machine Board)"
-			build_path = /obj/machinery/atmospherics/components/unary/thermomachine/freezer
-		else
-			name = "Heater (Machine Board)"
-			build_path = /obj/machinery/atmospherics/components/unary/thermomachine/heater
+/obj/item/weapon/circuitboard/machine/thermomachine/New()
```
Comment (Cyberboss) [inline code/modules/atmospherics/machinery/components/unary_devices/thermomachine.dm]: Initialize

File: code/modules/atmospherics/machinery/components/unary_devices/thermomachine.dm [hunk 2]
```diff
@@ -32,18 +32,17 @@
...
+	..()
```
Comment (Cyberboss) [inline code/modules/atmospherics/machinery/components/unary_devices/thermomachine.dm]: . = ..()

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**